app/controllers/apicasso/apidocs_controller.rb

```diff
@@ -8,6 +8,7 @@ class ApidocsController < Apicasso::ApplicationController
 
     include Swagger::Blocks
 
+    # Definitions of Application
```

Comment:
Improve description of "Definitions of Application", what is being defined and for what purpose?